packages/dds/tree/src/feature-libraries/sequence-field/index.ts

```diff
@@ -7,11 +7,12 @@ export {
     Attach,
     Changeset,
     ClientId,
+    Delete,
     Detach,
     Effects,
     GapCount,
     HasChanges,
-    HasOpId,
+    HasMoveId as HasOpId,
```

Comment:
VSCode "helped" you out.
```suggestion
    HasMoveId,
```

Reply:
Fixed